### ISSUE TITLE
RP AJ does not find bootable systems on external SSD, fixed in AK

### ISSUE THREAD

**Issue by MarioG-X** [dakanji/RefindPlus#86]
### RefindPlus Version

v0.13.2.AJ Release

### Device Type

Apple Mac

### Problem Description

Just an FYI and you may close this. I had an external drive that had Linux and macOS bootable systems. RP AJ did not show any of them. However, the strange thing is that a different non-SSD disk had no problem. RP AJ could boot systems on them. I don't think being an SSD has anything to do with the issue. The two are organized quite different but both have the same APFS and BTRFS filesystems. I ran the following tests:

* RP AJ finds all systems on the non-SSD but nothing on the SSD. The activity light seems RP does not look at the disk.
* Refind original finds all systems on both disks
* Using the Option key at boot the Mac finds all macOS systems
* Using RP X178 you sent for another issue, it works fine.

Again, I am sending in case you were not aware of this issue, but it's fixed in X178, looking forward to AK. Thank you!

### Problem Point

Before starting a loader or tool

### Affected Items

Other Item

### Debug Log and (Optional) Screenshots

No logs needed for solved issue

**Comment by dakanji** [comment]
Thanks for reporting.

BTW, can you help test the attached btrfs driver?
[btrfs_x64.zip](https://github.com/dakanji/RefindPlus/files/7610660/btrfs_x64.zip)

Building this driver was excluded from RP due to it causing the whole build to fail.
This has been fixed by @joevt but need to test the driver (doen't need to be by RP) to make sure it works as it should.
I believe you have OpenSuse instances that use btrfs.

**Comment by startergo** [comment]
As a side note my Opensuse (btrfs setup) boots without btrfs.efi, so when you test try without it first.

**Comment by MarioG-X** [comment]
Results:
Booting without a btrfs_64.efi: The two Opensuse btrfs systems do NOT display in the boot menu so NOGO.
Booting with the btrfs_64.efi from the zip file above works fine. Openeuse fully booted.
